xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -123,14 +132,64 @@ bool CVAAPIContext::EnsureContext(CVAAPIContext **ctx, CDecoder *decoder)
   return true;
 }
 
-bool CVAAPIContext::CreateContext()
+namespace
+{
+VADisplay FindValidDRMVaDisplayFromRenderNode()
 {
-  { CSingleLock lock(g_graphicsContext);
-    if (!m_X11dpy)
-      m_X11dpy = XOpenDisplay(NULL);
```

Comment:
I removed this, because it wasnt referenced anywhere, but I can put it back (not sure if there were plans for it later). Ill add back!